apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingStatusService.kt

```diff
@@ -201,20 +201,22 @@ class BehandlingStatusServiceImpl(
         if (featureToggleService.isEnabled(GenerellBehandlingToggle.KanBrukeGenerellBehandlingToggle, false)) {
             if (behandling.type == BehandlingType.FØRSTEGANGSBEHANDLING) {
                 if (behandling.boddEllerArbeidetUtlandet?.skalSendeKravpakke == true) {
+                    val opprettetBehandling =
+                        generellBehandlingService.opprettBehandling(
+                            GenerellBehandling.opprettUtland(
+                                GenerellBehandling.GenerellBehandlingType.UTLAND,
+                                behandling.sak.id,
+                                behandling.id,
+                            ),
+                        )
                     val oppgaveUtland =
                         oppgaveService.opprettNyOppgaveMedSakOgReferanse(
-                            behandling.id.toString(),
+                            opprettetBehandling.id.toString(),
                             behandling.sak.id,
-                            OppgaveKilde.BEHANDLING,
+                            OppgaveKilde.GENERELL_BEHANDLING,
                             OppgaveType.UTLAND,
                             null,
```

Comment:
Burde ansvaret for å opprette en oppgave for en generell behandling heller ligge i generellBehandlingService?

Reply:
Ja kanskje det er lurt

Reply:
Sjekker opp og ser hva som er smuudest

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2277/commits/3d56a0e56dfaf43b0e838bd0f3ac5b2d96aefd6c